edraak.mk

```diff
@@ -9,10 +9,10 @@ edraak.build.all:
 	$(MAKE) edraak.build.marketing
 
 edraak.build.programs:
-	docker build -t edraak-devstack-programs -f ../edraak-programs/Dockerfile ../edraak-programs
+	docker build -t eu.gcr.io/openedx-231314/edraak/progs -f ../edraak-programs/Dockerfile ../edraak-programs
```

Comment:
Great. Let's undo the edraak hacks on `make pull` and make it look like this:

```
pull: ## Update Docker images
	docker-compose pull --parallel
```

Reply:
done